### DISCUSSION TITLE
lolz!

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#304] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
imagine having lightspeed instead of dyknow when dyknow is 10 million times easier and more annoying to bypass!!!!
lolz!!!

**Comment by bartholemuew** [comment, marked as answer]
what purpose does this have

**Reply by rwg100** [reply]
to be cooler lolz!!!!!!!!!!!!!!!

**Reply by PickleIsDaBest** [reply]
lmao

**Comment by Aka-but-nice** [comment]
Bro, I wish I had a blocker that I could catch slipping securly ain't easy to bypass!
